content/posts/manage-list-feature-flags-with-file/index.md

```diff
@@ -0,0 +1,229 @@
++++
+date = "2021-12-29T14:28:23+07:00"
+author = "nhattien.ho"
+description = "Manage list feature flags with file yaml/json."
+title = "Create feature flags using file in Unleash"
+categories = ["Feature toggle", "Testing"]
+tags = ["Feature toggle", "Unleash", "Node.js"]
+slug = "manage-list-flags-with-file"
++++
+
+In this blog, we will learn how to create a list of feature toggles in Unleash using file. First, let's learn some concept of feature toggle and Unleash 
+#### What is feature toggle (feature flag)?
+Feature toggle (Feature flag) is a technique that allows you to disable some functionality of your application, through settings, without having to deploy new code.
+#### What is Unleash?
+[Unleash](https://docs.getunleash.io/) is a feature toggle system, that gives you a great overview of all feature toggles across all your applications and services. It comes with official client implementations for Java, Node.js, Go, Ruby, Python and .Net.
+#### Why do we use file to create feature flags instead of Admin UI?
+In Manabie, we manage the addition and removal of feature flags with pull request, so we need a solution to manage the list of feature flags by file. Developers can see and change the list of feature flags by modifying this file, don't need to access the admin page. The list of feature flags will be updated after we deploy new code backend.
```

Comment:
At Manabie, we manage the addition and removal of feature flags with pull request, so we need a solution to manage the list of feature flags by file. Developers can see and change the list of feature flags by modifying this file, don't need to access the admin page. The list of feature flags will be updated after we deploy new backend code.